doc/user_guide/color_usage.rst

```diff
@@ -0,0 +1,1525 @@
+
+   * - .. code-block:: none
+    
+         tol_nightfall
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("tol_nightfall", tol_schemes, cvd=True, continuous=True, grayscale=True)
+
+   * - .. code-block:: none
+    
+         tol_iridescent
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("tol_iridescent", tol_schemes, cvd=True, continuous=True, grayscale=True)
+
+   * - .. code-block:: none
+    
+         tol_incandescent
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("tol_incandescent", tol_schemes, cvd=True, continuous=True, grayscale=True)
+
+Expressions and colors
+~~~~~~~~~~~~~~~~~~~~~~
+
+We can make use of the measured values regarding colors in a visualization.
+In the following example we will make use of the ``lumniance`` to detect if 
+the color of a text overlay on top of a bar should be colored ``black`` or
+``white``. The luminance describes the brightnes, normalized to 0 for darkest
+black and 1 for lightest white.
+
+In the following chart we have a sorted bar chart where the text overlay describes
+the sum of yield as is expressed on the ``x`` encoding channel.
+
+.. altair-plot::
+
+    import altair as alt
+    from vega_datasets import data
+
+    source = data.barley()
+
+    base = alt.Chart(source).encode(
+        x=alt.X('sum(yield):Q').stack('zero'),
+        y=alt.Y('site:O').sort('-x'),
+        text=alt.Text('sum(yield):Q', format='.0f')
+    )
+
+    bars = base.mark_bar().encode(color='sum(yield):Q')
+    text = base.mark_text(align='right', dx=-3, color='black')
+
+    bars + text
+
+As you can see, when the bar becomes darker, the text overlay becomes less visible.
+
+At this moment we can make use of the measured luminance to decide if the text overlay
+should be colored ``black`` or ``white``.
+
+.. altair-plot::
+
+    import altair as alt
+    from vega_datasets import data
+
+    source = data.barley()
+
+    base = alt.Chart(source).encode(
+        x=alt.X('sum(yield):Q').stack('zero'),
+        y=alt.Y('site:O').sort('-x'),
+        text=alt.Text('sum(yield):Q', format='.0f')
+    )
+
+    bars = base.mark_bar(
+        tooltip=alt.expr("luminance(scale('color', datum.sum_yield))")
+    ).encode(
+        color='sum(yield):Q'
+    )
+
+    text = base.mark_text(
+        align='right', 
+        dx=-3,
+        color=alt.expr("luminance(scale('color', datum.sum_yield)) > 0.5 ? 'black' : 'white'")
+    )
+
+    bars + text
+
+The lighter the bar, the higher the luminance. If the bar is light
+we like a text overlay that is black. The darker the bar, the lower
+the luminance. If the bar is dark, we like a text overlay that is white.
+
+In the expression above we have written this as a predicate. The text
+appear ``black`` if the luminance is above ``0.5`` and ``white`` when
+the luminance is below ``0.5``. The luminance is computed using the
```

Comment:
```suggestion
        color=alt.expr("luminance(scale('color', datum.sum_yield)) > 0.408 ? 'black' : 'white'")
    )

    bars + text

The lighter the bar, the higher the luminance. If the bar is light
we like a text overlay that is black. The darker the bar, the lower
the luminance. If the bar is dark, we like a text overlay that is white.

In the expression above we have written this as a predicate. The text
appear ``black`` if the luminance is above ``0.408`` and ``white`` when
the luminance is below ``0.408``. The luminance is computed using the
```

This section is great! I love that expression can be used for this! The reason for the change I suggest here is to stay consistent with what is used in seaborn, which is also what I chose when adding this to the text of pandas heatmaps. We could add a note about this? (I was trying to look up the reason for picking this exact number in the first place, but based on [my comment here ](https://github.com/pandas-dev/pandas/pull/21263/files#r193573758)I don't think I ever knew =) )